libs/sdk-ui-filters/src/AttributeFilter@next/Components/AttributeFilterDropdownBody.tsx

```diff
@@ -0,0 +1,41 @@
+// (C) 2022 GoodData Corporation
+import React, { useCallback } from "react";
+import {
+    AttributeFilterDropdownContent,
+    IAttributeDropdownBodyPropsNoCallbacks,
+} from "./AttributeFilterDropdownContent";
+import { useAttributeFilterComponentsContext } from "../Context/AttributeFilterComponentsContext";
+
+interface IAttributeFilterDropdownBodyProps {
+    allElementsFiltered: boolean;
```

Comment:
maybe we should align it with `hasNoData` ? (eg `hasNoMatchingData`)

Reply:
renamed